sal_interfaces/ATMCS/ATMCS_Commands.xml

```diff
@@ -58,12 +58,12 @@
             <EFDB_Name>elevationVelocity</EFDB_Name>
             <Description>Mount elevation angle velocity at the specified time.</Description>
             <IDL_Type>double</IDL_Type>
-            <Units>deg/sec</Units>
+            <Units>deg / s</Units>
             <Count>1</Count>
         </item>
         <item>
             <EFDB_Name>nasmyth1RotatorAngle</EFDB_Name>
-            <Description>Nasmyth 1 rotator angle at the specified time. The allowed range is the software limits (L1), approximately -165 to 165. Set to nan if not using this axis.</Description>
+            <Description>Nasmyth 1 rotator angle at the specified time. The allowed range is the software limits (L1), approximately -170 to 170. Outside this range the velocity will be severly limited. Software position limits are -175 to 175. Set to nan if not using this axis.</Description>
```

Comment:
I don't think `nan` is a valid entry. I imagine it should be zero (the default value).